invenio_app_rdm/theme/views.py

```diff
@@ -75,6 +90,34 @@ def init_menu():
             order=1,
         )
 
+        current_menu.submenu("cog.record").register(
+            "administration.Records",
+            _("Manage record"),
+            order=1,
+            endpoint_arguments_constructor=lambda: {
+                "q": f"id:{request.view_args.get('pid_value')}"
+            },
+            visible_when=_can_manage_record,
+        )
+
+        current_menu.submenu("cog.user").register(
+            "administration.users",
+            _("Manage user"),
+            order=2,
+            endpoint_arguments_constructor=lambda: {"q": f"id:{current_user.id}"},
```

Comment:
Comment for reviewers, in the issue is specified to use the email. However if the preferences of the visibility are not set to public we cannot find the user by using the email, therefore I added the id that will always find the user (when searching as admin)

Reply:
We can see all the users in the admin page but we cannot search by email unless they have set the Profile visibility to Public.

## List all users
![Screenshot from 2023-10-05 10-41-33](https://github.com/inveniosoftware/invenio-app-rdm/assets/15194802/560d0301-522b-4c91-8cb2-5991a4509f89)

## Search for user with Public profile
![Screenshot from 2023-10-05 10-41-53](https://github.com/inveniosoftware/invenio-app-rdm/assets/15194802/3bbc0848-3429-4067-92db-fab4563f126b)

## Search for user with hidden profile
![Screenshot from 2023-10-05 10-42-02](https://github.com/inveniosoftware/invenio-app-rdm/assets/15194802/24434e5b-c078-4ae9-824e-4918485a0973)

Reply:
Nice finding! thanks for checking this. 

in this case I think we are just missing User manager generator (or permission?) in a right place for searching emails:

https://github.com/inveniosoftware/invenio-users-resources/blob/master/invenio_users_resources/services/permissions.py#L39

due to https://github.com/inveniosoftware/invenio-users-resources/blob/dbb9126b725b26501f4921a7d73fe8e15a3d2ea8/invenio_users_resources/services/schemas.py#L48

I think it is better to handle it by permissions rather than trying to circumvent it via ID. maybe makes sense to discuss during the standup?

Reply:
The email is not indexed if the user profile is set to hidden, therefore no one can search for it as it's not available, agreed over [chat](https://discord.com/channels/692989811736182844/1092367535141289984/1159802012288958494) that we will use the username instead, which unique and indexed all the time and only admins can search for it.